x-pack/plugins/secops/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -88,14 +88,14 @@ const HostsTableComponent = pure<HostsTableProps>(
 );
 
 const mapStateToProps = (state: State) => {
-  const limit = defaultTo(2, hostsSelector(state));
+  const limit = defaultTo(2, hostsLimitSelector(state));
   return { limit };
```

Comment:
Optiopnal: I think you can do a simple `const  mapStateToProps = (state: State) => hostsSelector(state)` at this point since the `defaultTo(` no longer is needed because of the init state?